htsinfer/cli.py

```diff
@@ -265,13 +268,7 @@ def main() -> None:
         # handle CLI args
         args = parse_args()
 
-        # set up logging
-        setup_logging(verbosity=args.verbosity)
-        LOGGER.info("Started HTSinfer...")
-        LOGGER.debug(f"CLI arguments: {args}")
-
-        # determine library metadata
-        hts_infer = HtsInfer(
+        arguments = Args(
```

Comment:
It would be nice to make use of `dest` in the CLI argument definitions to define the names for each variable rather than mapping the more descriptive names from the CLI args to the more concise ones used interally. That would allow us to pass arguments to `Args` through unpacking `args`.

For example:

```py
    parser.add_argument(
        "--output-directory",
        dest="out_dir",  # <-- this line is new
        default=Path.cwd() / 'results_htsinfer',
        type=lambda p: Path(p).absolute(),
        metavar="PATH",
        help="path to directory where output is written to",
    )
```

Now, if we do that for all of the args, we could do something like this (untested):

```py
args_dict: Dict = vars(args)
del args_dict["paths"]
arguments = Args(path_1=args.paths[0], path_2=args.paths[1], **args_dict)
```

Or even more concisely, we could do:

```py
config = Config(
   args=Args(path_1=args.paths[0], path_2=args.paths[1], **args_dict),
   results=Results(),
)
```